lib/trailblazer/operation/public_call.rb

```diff
@@ -49,10 +64,5 @@ def options_for_public_call(*args)
     def self.options_for_public_call(options, **flow_options)
       Trailblazer::Context(options, {}, flow_options[:context_options])
     end
-
-    # @semi=public
-    def flow_options
```

Comment:
@apotonick Let me know if this method removal can be accompanied in case you've overridden it.